### DISCUSSION TITLE
One of my motor flies without me controlling it

### DISCUSSION THREAD

**Discussion by akshaysunkara** [bitcraze/discussions#1382] [Q&A]
As soon as I plug the motor in, the motor starts without me controlling it. All the other motors are just fine.

**Comment by gemenerik** [comment]
That is odd, do you see the same behavior with stock firmware and no running apps?

**Reply by akshaysunkara** [reply]
Yes, I do

**Reply by akshaysunkara** [reply]
It happens as soon as I plug it in, no software or anything

**Reply by gemenerik** [reply]
Can you try flashing the stock firmware?
